browser/flag_descriptions.h

```diff
@@ -0,0 +1,150 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_FLAG_DESCRIPTIONS_H_
+#define BRAVE_BROWSER_FLAG_DESCRIPTIONS_H_
+
+#include "brave/components/brave_rewards/common/buildflags/buildflags.h"
+
+namespace flag_descriptions {
```

Comment:
current impl seems ok, just one note: ```flag_descriptions.h``` can be included directly into ```browser/about_flags.cc``` and ```chromium_src/chrome/browser/flag_descriptions.h``` can be removed.

But what about putting all this strings into ```browser/about_flags.cc```? Everything will be in one place and it can look like that:
```
#include "..."
...

namespace flag_descriptions {
namespace {

constexpr char kBraveAdblockCnameUncloakingName[] = "Enable CNAME uncloaking";
constexpr char kBraveAdblockCnameUncloakingDescription[] =
    "Take DNS CNAME records into account when making network request blocking "
    "decisions.";
...

}  // namespace
}  // namespace flag_descriptions

using brave_shields::features::kBraveAdblockCnameUncloaking;
...

#if BUILDFLAG(ENABLE_BRAVE_VPN)
#define BRAVE_VPN_FEATURE_ENTRIES                         \
    {kBraveVPNFeatureInternalName,                        \
     flag_descriptions::kBraveVPNName,                    \
     flag_descriptions::kBraveVPNDescription,             \
     kOsMac | kOsWin | kOsAndroid,                        \
     FEATURE_VALUE_TYPE(brave_vpn::features::kBraveVPN)},
#else
#define BRAVE_VPN_FEATURE_ENTRIES
#endif
...
```

Reply:
Sounds good to me too, updated PR once again!